h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -233,16 +235,18 @@ private void writeModelInfo() throws IOException {
     finishWritingTextFile();
   }
 
+  private static final Pattern QUOTE_PATTERN = Pattern.compile("[\"]{1}"); // Escape quotes
```

Comment:
Not needed anymore

Reply:
Removed. As well as the line down below this.